docs/01-Using-Fleet/standard-query-library/standard-query-library.yml

```diff
@@ -751,3 +751,51 @@ spec:
   tags: compliance, hardening, built-in
   platform: darwin
   contributors: GuillaumeRoss
+---
+apiVersion: v1
+kind: policy
+spec:
+  name: Operating system up to date (macOS)
+  query: SELECT 1 FROM os_version WHERE version >= '12.5.1';
+  description: "Checks that the operating system is up to date."
+  resolution: "Go to the Software Update section of System Preferences and install the latest updates."
+  platforms: macOS
+  tags: compliance, CIS, template, CIS1.1
+  platform: darwin
+  contributors: GuillaumeRoss
+---
+apiVersion: v1
+kind: policy
+spec:
+  name: Automatic updates enabled (macOS)
+  query: SELECT 1 FROM managed_policies WHERE domain='com.apple.SoftwareUpdate' AND name='AutomaticCheckEnabled' AND value=1 LIMIT 1;
+  description: "Checks that the operating system is configured to automatically checks for updates."
+  resolution: "Go to the Software Update section of System Preferences and install the latest updates."
+  platforms: macOS
+  tags: compliance, CIS, CIS1.2
+  platform: darwin
+  contributors: GuillaumeRoss
+---
+apiVersion: v1
+kind: policy
+spec:
+  name: Automatic updates enabled (macOS)
+  query: SELECT 1 FROM managed_policies WHERE domain='com.apple.SoftwareUpdate' AND name='AutomaticCheckEnabled' AND value=1 LIMIT 1;
+  description: "Checks that the operating system is configured to automatically download updates."
+  resolution: "Contact your IT administrator to ensure your Mac is receiving a profile that enables automatic update downloads."
+  platforms: macOS
+  tags: compliance, CIS, CIS1.3
+  platform: darwin
+  contributors: GuillaumeRoss
+---
+apiVersion: v1
+kind: policy
+spec:
+  name: Installation of application updates is enabled (macOS)
+  query: SELECT 1 FROM managed_policies WHERE domain='com.apple.SoftwareUpdate' AND name='AutomaticCheckEnabled' AND value=1 LIMIT 1;
```

Comment:
This `query` looks the same as the `query` for the "Automatic updates enabled (macOS)" policy. Is that intentional?